gazebo/Dockerfile

```diff
@@ -67,11 +67,33 @@ COPY ./resources /usr/local/share/pypi-confs/resources
 RUN /usr/local/bin/setup-pypi-env.sh INT
 
 ## Install messaging broker for async calls
-RUN apt-get install rabbitmq-server
+RUN apt-get update && \
```

Comment:
fix reported syntax errors on the Dockerfile,  
also maybe the folder `gazebo` should be renamed `docker` now